### ISSUE TITLE
(C1-2569-2) 

### ISSUE THREAD

**Issue by voild980** [xerpi/vita-udcd-uvc#67]
It gives me this error at 1280x720. 960x544 works normal.

**Comment by xerpi** [comment]
Probably means at 1280x720 the allocated framebuffer is too big and the game doesn't see enough free memory.

**Comment by voild980** [comment]
There's nothing we can do?

**Comment by cuevavirus** [comment]
There's probably a memory leak somewhere in the Sony kernel code or in taihen. This is a problem that can occur even without udcd-uvc and is exacerbated by high kernel memory allocation. It can manifest in a few ways but most commonly SceProcessmgr will fail to allocate PLS when starting up an application. It doesn't always happen but if it happens the only thing you can do is restart and it'll go away.

**Comment by voild980** [comment]
Yeah, I did it. Now, instead of showing that error code, my Vita just hangs up.

**Comment by xerpi** [comment]
> Yeah, I did it. Now, instead of showing that error code, my Vita just hangs up.

Init with plugins disabled by pressing L and try with this version: https://github.com/xerpi/vita-udcd-uvc/issues/63#issuecomment-657028268

**Comment by voild980** [comment]
Did not work...

**Comment by LiEnby** [comment]
Open email application, close it, then open whatever game (i dont know why this works)
